packages/react-magma-docs/src/data/properties.json

```diff
@@ -6213,6 +6240,14 @@
         },
         "deprecated": false
       },
+      "renderItem": {
```

Comment:
Looks like the old prop name is being generated as an option here... is this correct, or should there be an update to the `components` prop documentation?

Reply:
should be taken out.

Reply:
I'm sorry, I was wrong. This is being auto-generated but not yet being used. I don't believe there is a way to stop generating some of the files that haven't had the typedoc info put in yet so you will see this, but the docs page does not use the generated table.